docs/edsl.html

```diff
@@ -54,7 +54,7 @@ <h3 id="sum-over-axis">Sum Over Axis</h3>
 \[
 \Large
   \textcolor{red}{O(n)}
-  \textcolor{green}{+=}
+  \textcolor{green}{\,{+}{=}}
```

Comment:
Not a big deal (can skip), but it looks like this needs an extra `\,` at the end for a bit more space, i.e. `\,{+}{=}\,`